### ISSUE TITLE
CDB lists doesn't allow colon characters in keys and values

### ISSUE THREAD

**Issue by chemamartinez** [wazuh/wazuh#1500]
The CDB list files are plain text files where each line has the format `key:value`.

This is why, it is not able to use colon into those fields like, for example, Windows paths: 

```
data_path:C:\ProgramData\test
```

It has to allow to scape those characters to differentiate them from the separator one.

**Comment by branchnetconsulting** [comment]
Thanks for opening this issue for me, Chema.  I do think it would be helpful to support colons existing inside of keys and/or values in CDBs.  Either some way of escaping individual colons, or perhaps some way of optionally surrounding the key and/or value by some additional delimiter like quotes are used in CSV files, might do the job.

**Comment by TJOSERAFAEL** [comment]
This PR https://github.com/wazuh/wazuh/pull/2046 solves the issue

**Comment by chemamartinez** [comment]
Hi @branchnetconsulting,

Let me close the issue since it has been solved and merged at #2046.

Now, you can specify keys and values into CDB lists surrounded by double quotes such as: `"my_key":"C:\Program Files\my-app"` and match them as follow:

```
<list field="test" lookup="match_key_value" check_value="C:\Program Files\my-app">etc/lists/test</list>
```

Regards.

**Comment by branchnetconsulting** [comment]
Awesome, thanks!  Example appreciated, too.

Kevin
